src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: 100px 10px 10px;
+  width: $monthWidth;
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: $boxSize;
+    width: $boxSize;
+    background-color: #eee;
+    border: 1px solid black;
+    transition: transform 0.5s;
+  }
+
+  @for $index from 1 through 31 {
+    &__day:nth-child(#{$index})::before {
+      content: "#{$index}";
+      font-size: 30px;
```

Comment:
Don't set such property in @for the cycle. You can set it once in the `html {}` selector